Plugins/Gerrit/FormGerritPublish.Designer.cs

```diff
@@ -73,102 +74,123 @@ private void InitializeComponent()
             this._NO_TRANSLATE_Topic.TabIndex = 6;
             // 
             // _NO_TRANSLATE_Reviewers
             // 
             this._NO_TRANSLATE_Reviewers.Anchor = ((System.Windows.Forms.AnchorStyles)(((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Left) 
             | System.Windows.Forms.AnchorStyles.Right)));
-            this._NO_TRANSLATE_Reviewers.Location = new System.Drawing.Point(101, 108);
+            this._NO_TRANSLATE_Reviewers.Location = new System.Drawing.Point(126, 135);
+            this._NO_TRANSLATE_Reviewers.Margin = new System.Windows.Forms.Padding(4);
             this._NO_TRANSLATE_Reviewers.Name = "_NO_TRANSLATE_Reviewers";
-            this._NO_TRANSLATE_Reviewers.Size = new System.Drawing.Size(522, 21);
+            this._NO_TRANSLATE_Reviewers.Size = new System.Drawing.Size(652, 22);
             this._NO_TRANSLATE_Reviewers.TabIndex = 8;
             // 
             // labelReviewers
             // 
             this.labelReviewers.AutoSize = true;
-            this.labelReviewers.Location = new System.Drawing.Point(13, 111);
+            this.labelReviewers.Location = new System.Drawing.Point(16, 139);
+            this.labelReviewers.Margin = new System.Windows.Forms.Padding(4, 0, 4, 0);
             this.labelReviewers.Name = "labelReviewers";
-            this.labelReviewers.Size = new System.Drawing.Size(69, 13);
+            this.labelReviewers.Size = new System.Drawing.Size(87, 17);
             this.labelReviewers.TabIndex = 7;
             this.labelReviewers.Text = "Reviewer(s):";
             // 
+            // PublishType
+            // 
+            this.PublishType.DisplayMember = "Key";
+            this.PublishType.FormattingEnabled = true;
+            this.PublishType.Location = new System.Drawing.Point(126, 172);
+            this.PublishType.Margin = new System.Windows.Forms.Padding(4);
+            this.PublishType.Name = "PublishType";
+            this.PublishType.Size = new System.Drawing.Size(121, 24);
+            this.PublishType.TabIndex = 11;
+            this.PublishType.ValueMember = "Value";
+            // 
+            // labelPublishType
+            // 
+            this.labelPublishType.AutoSize = true;
+            this.labelPublishType.Location = new System.Drawing.Point(16, 175);
+            this.labelPublishType.Margin = new System.Windows.Forms.Padding(4, 0, 4, 0);
+            this.labelPublishType.Name = "labelPublishType";
+            this.labelPublishType.Size = new System.Drawing.Size(90, 17);
+            this.labelPublishType.TabIndex = 12;
+            this.labelPublishType.Text = "Publish Type";
+            // 
             // FormGerritPublish
             // 
             this.AcceptButton = this.Publish;
-            this.AutoScaleDimensions = new System.Drawing.SizeF(96F, 96F);
+            this.AutoScaleDimensions = new System.Drawing.SizeF(120F, 120F);
```

Comment:
⚠️  This is not good. Please update the designer at 100% scaling.
The easiest is to do with `devenv /noscaling` (writing from memory)

https://docs.microsoft.com/en-us/dotnet/framework/winforms/disable-dpi-awareness-visual-studio#restart-visual-studio-as-a-dpi-unaware-process
![image](https://user-images.githubusercontent.com/4403806/59270744-9c314080-8c95-11e9-81a8-dd51ae677334.png)

Reply:
Thank you. Will do.

Reply:
Correction the correct switch is `/noscale`
![image](https://user-images.githubusercontent.com/4403806/59431442-4d67e000-8e28-11e9-91e0-029b2dbc8e9f.png)

Reply:
I have fixed the designer with `/noscale`